test/post-data-concat-test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var fs = require('fs');
+var soap = require('..');
+var assert = require('assert');
+var http = require('http');
+
+
+describe('post data concat test', function () {
+    it('should consider the situation about multi-byte character between two tcp packets', function (done) {
+        var check = function (a, b) {
+            if (a && b) {
+                assert(a === b);
+                done();
+            }
+        };
+
+        var wsdl = 'test/wsdl/default_namespace.wsdl';
+        var server = http.createServer(function (req, res) { });
+        var xml = fs.readFileSync(wsdl, 'utf8');
+        var service = {
+            MyService: {
+                MyServicePort: {
+                    MyOperation: function (arg) {
+                        console.log(arg);
+                        check(arg, postdata);
+                        return "0";
+                    }
+                }
+            }
+        };
+
+        server.listen(51515);
```

Comment:
can you move the server declaration/initialization to a `before` block, then in an `after` block tear it down?

Reply:
done.pls check if it is right.